releasenotes/notes/0.18/TensoredMeasFitter-d327f0e2d736efdd.yaml

```
@@ -0,0 +1,14 @@
+---
+features:
+  - |
+    The ``measurement_error_mitgation`` kwarg for the
+    :class:`~qiskit.utils.QuantumInstance` constructor can now be set to the
+    :class:`~qiskit.ignis.mitigation.TensoredMeasFitter` class from
+    qiskit-ignis in addition to
+    :class:`~qiskit.ignis.mitigation.CompleteMeasFitter` that was already
+    supported. If you use :class:`~qiskit.ignis.mitigation.TensoredMeasFitter`
+    you will need to also set the new ``mit_pattern`` kwarg to specify the
```

I think it's optional and per default does measurement mitigation on each qubit individually
```suggestion
    you will also be able to set the new ``mit_pattern`` kwarg to specify the
```